### PR TITLE
Added additional tests for 920120

### PR DESCRIPTION
**Pull request by theseion**
Related to https://github.com/coreruleset/coreruleset/pull/2360

This is PR 1 of 4 that replace #2360.

### CODE REVIEW

File: tests/regression/tests/REQUEST-920-PROTOCOL-ENFORCEMENT/920120.yaml
Comment (fzipi) [inline tests/regression/tests/REQUEST-920-PROTOCOL-ENFORCEMENT/920120.yaml:889]: This looks like a positive test, right? Probably comment stayed because of copy&paste

**Review by fzipi — CHANGES_REQUESTED** [review]
Other than comment, looks good. After changing ping me so it is ready to go.

File: tests/regression/tests/REQUEST-920-PROTOCOL-ENFORCEMENT/920120.yaml
Comment (theseion) [reply tests/regression/tests/REQUEST-920-PROTOCOL-ENFORCEMENT/920120.yaml]: Yes, you're absolutely right. I've fixed the comments.

**Review by fzipi — APPROVED** [review]
LGTM now!

**Comment by theseion** [comment]
I don't think we need a second review here, it's a simple change. Merging now.

**Pull request merged**
